spec/state_machines/overturned_sar_event_spec.rb

```diff
@@ -95,6 +133,29 @@ def all_cases
         [:disclosure_bmt, :ot_ico_sar_noff_awresp],
         [:disclosure_bmt, :ot_ico_sar_noff_draft],
         [:disclosure_bmt, :ot_ico_sar_noff_closed],
+        [:disclosure_bmt, :ot_ico_sar_noff_trig_awresp],
+        [:disclosure_bmt, :ot_ico_sar_noff_trig_awresp_accepted],
+        [:disclosure_bmt, :ot_ico_sar_noff_trig_draft],
+        [:disclosure_bmt, :ot_ico_sar_noff_trig_draft_accepted],
+        [:disclosure_bmt, :ot_ico_sar_noff_pdacu],
+
+        [:disclosure_specialist, :ot_ico_sar_noff_pdacu],
+
+        [:disclosure_specialist_coworker, :ot_ico_sar_noff_pdacu],
+
+        [:another_disclosure_specialist, :ot_ico_sar_noff_pdacu],
+
+        [:responder, :ot_ico_sar_noff_pdacu],
+
+        [:another_responder_in_same_team, :ot_ico_sar_noff_pdacu],
+
+        [:another_responder_in_diff_team, :ot_ico_sar_noff_pdacu],
+
+        [:press_officer, :ot_ico_sar_noff_pdacu],
```

Comment:
should press and private officers be able to link cases, since it's a SAR case I would expect that they wouldn't be able to view it